src/Cli/dotnet/commands/dotnet-check-update/xlf/LocalizableStrings.ko.xlf

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="ko" original="../LocalizableStrings.resx">
+    <body>
+      <trans-unit id="AppFullName">
+        <source>.NET Check Update Command</source>
+        <target state="needs-review-translation">.NET 정리 명령</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="AppDescription">
+        <source>Check for SDK and runtime updates.</source>
+        <target state="needs-review-translation">이전에 생성된 빌드 출력을 정리하는 명령입니다.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="BundleUpToDateMessage">
+        <source>Up to date.</source>
+        <target state="new">Up to date.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="MaintenanceMessage">
+        <source>.NET {0} is going out of support soon.</source>
+        <target state="new">.NET {0} is going out of support soon.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="NewFeatureBandMessage">
+        <source>Try out the newest .NET SDK features with .NET {0}.</source>
+        <target state="new">Try out the newest .NET SDK features with .NET {0}.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="NewPatchAvaliableMessage">
+        <source>Patch {0} is available.</source>
+        <target state="new">Patch {0} is available.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="OutOfSupportMessage">
+        <source>.NET {0} is out of support.</source>
+        <target state="new">.NET {0} is out of support.</target>
```

Comment:
Should these message perhaps include a link to https://dotnet.microsoft.com/platform/support/policy/  (I'm unsure whether we have an aka.ms link for the page)